_inc/client/plans/style.scss

```diff
@@ -221,3 +221,20 @@ $plan-features-sidebar-width: 272px;
 .plans-mobile-notice.dops-card h2 {
 	margin-top: 0;
 }
+.plan-grind-skeletons {
```

Comment:
We shouldn't be displaying this placeholder for mobile - for mobile we display a button that links to the .com plans grid: 

![](https://cldup.com/JHskQsYTcj.png)

Reply:
Done!